### PR TITLE
feat: add support for locally installing packages to the python backend

### PR DESCRIPTION
**Pull request by sergiolaverde0**
Draft of the changes on the python service to enable installing models not baked into the image itself to a mounted volume.

### CODE REVIEW

**Comment by simjanos-dev** [comment]
I've read our comments back, but I could not find the latest thread where we talked about this. Is this PR finished, or is there still a problem with it?

If it is finished, I'll try to focus on writing my part for it. I've been working on some random stuff lately.

**Comment by sergiolaverde0** [comment]
It is finished. I made it grab the language to install from the header instead of grabbing it from the body like your code does to make it easier to test for me but I can change it to make it consistent, let me know.

I will address the merge conflicts once I get home, so late night for you.

**Comment by simjanos-dev** [comment]
> I will address the merge conflicts once I get home, so late night for you.

It's just an additional language in the dockerfile, I can fix it.

File: docker/PythonDockerfile [hunk 1]
```diff
@@ -37,6 +37,8 @@ RUN python3 -m spacy download de_core_news_sm \
     && python3 -m spacy download fr_core_news_sm \
     && python3 -m spacy download it_core_news_sm \
     && python3 -m spacy download sv_core_news_sm \
+    && python3 -m spacy download uk_core_news_sm \
+    && python3 -m spacy download en_core_web_sm \
+    && python3 -m spacy download el_core_news_sm \
     && python3 -m spacy download xx_ent_wiki_sm
 
-COPY ./tools /app
```
Comment (simjanos-dev) [inline docker/PythonDockerfile:49]: I think this one was an accident.

Comment (sergiolaverde0) [reply docker/PythonDockerfile:49]: Yep, that merge had some side-effects. Fixed.

**Comment by simjanos-dev** [comment]
I wanted to test v0.10, and start working on this in the meantime, but I have internet outages every few minutes. It might take a day or two before I'll be able to work on it.

If you are looking for something linguacafe related to work on, I am planning on adding Vietnamese and Thai soon to spacy when this feature is ready. I was experimenting with Latin as well, but could not get it to work (currently it uses multilingual model).

(Not trying to nudge/pressure you working on something, just a suggestion. I'm not very good at communicating.)

**Comment by sergiolaverde0** [comment]
Yesterday while solving that conflict I thought of asking if I should add Vietnamese too, but decided against it in order not to make this PR bigger than it already is. So I will add those on a different branch and open a new PR after this one gets merged.

It's fine, you're not as bad as communicating as you seem to believe.

**Comment by simjanos-dev** [comment]
I merge this in, and will start working on the UI/Laravel part probably today. I won't touch the Python part, so if you will have a new PR, there should not be any conflict.

**Pull request merged**